specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2024-11-01-preview/containerInstance.json

```diff
@@ -0,0 +1,2666 @@
+              "$ref": "#/definitions/ContainerState"
+            },
+            "previousState": {
+              "readOnly": true,
+              "description": "The previous state of the init container.",
+              "$ref": "#/definitions/ContainerState"
+            },
+            "events": {
+              "readOnly": true,
+              "description": "The events of the init container.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Event"
+              },
+              "x-ms-identifiers": [
+                "name",
+                "type"
+              ]
+            }
+          }
+        },
+        "volumeMounts": {
+          "description": "The volume mounts available to the init container.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VolumeMount"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "securityContext": {
+          "description": "The container security properties.",
+          "$ref": "#/definitions/SecurityContextDefinition"
+        }
+      }
+    },
+    "CloudError": {
+      "x-ms-external": true,
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/CloudErrorBody"
+        }
+      },
+      "description": "An error response from the Container Instance service."
+    },
+    "CloudErrorBody": {
+      "x-ms-external": true,
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "An identifier for the error. Codes are invariant and are intended to be consumed programmatically."
+        },
+        "message": {
+          "type": "string",
+          "description": "A message describing the error, intended to be suitable for display in a user interface."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the particular error. For example, the name of the property in error."
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudErrorBody"
+          },
+          "x-ms-identifiers": [
+            "message",
+            "target"
+          ],
+          "description": "A list of additional details about the error."
+        }
+      },
+      "description": "An error response from the Container Instance service."
+    },
+    "NetworkDependenciesResponse": {
+      "description": "Response for network dependencies, always empty list.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "ConfidentialComputeProperties": {
+      "description": "The properties for confidential container group",
+      "type": "object",
+      "properties": {
+        "ccePolicy": {
+          "type": "string",
+          "description": "The base64 encoded confidential compute enforcement policy"
+        }
+      }
+    },
+    "IdentityAcls": {
```

Comment:
>IdentityAcls [](http://example.com/codeflow?start=5&length=12)

so this has 2 levels of acl information, one that is default and one that is for each identity then ?

Reply:
Yes, exactly, there is a list of identities with specific access levels and a default access level that will be used for identities not appearing in that list. So you can see it as a default access level that is modified for a list of identities.